src/portal/src/app/project/p2p-provider/task-list/task-list.component.ts

```diff
@@ -0,0 +1,223 @@
+    }
+    if (this.executionId) {
+      this.preheatService.GetExecution({
+        projectName: this.projectName,
+        preheatPolicyName: this.preheatPolicyName,
+        executionId: +this.executionId
+      }).pipe(finalize(() => (this.inProgress = false)))
+        .subscribe(res => {
+          this.execution = res;
+            if (!this.execution || this.p2pProviderService.willChangStatus(this.execution.status)) {
+              setTimeout(() => {
+                this.getExecutionDetail(false);
+              }, TIME_OUT);
+            }
+        },
+        error => {
+          this.messageHandlerService.error(error);
+        });
+    }
+  }
+  trigger(): string {
+    return this.execution && this.execution.trigger
+      ? this.execution.trigger
+      : "";
+  }
+
+  startTime(): string {
+    return this.execution && this.execution.start_time
+      ? this.execution.start_time
+      : null;
+  }
+
+  successNum(): number {
+    if (this.execution && this.execution.metrics) {
+      return this.execution.metrics.success_task_count;
+    }
+    return 0;
+  }
+
+  failedNum(): number {
+    if (this.execution && this.execution.metrics) {
+      return this.execution.metrics.error_task_count;
+    }
+    return 0;
+  }
+
+  progressNum(): number {
+    if (this.execution && this.execution.metrics) {
+      return (this.execution.metrics.pending_task_count ? this.execution.metrics.pending_task_count : 0)
+        + (this.execution.metrics.running_task_count ? this.execution.metrics.running_task_count : 0)
+        + (this.execution.metrics.scheduled_task_count ? this.execution.metrics.scheduled_task_count : 0);
+    }
+    return 0;
+  }
+
+  stoppedNum(): number {
+    if (this.execution && this.execution.metrics) {
+      return this.execution.metrics.stopped_task_count;
+    }
+    return 0;
+  }
+
+  stopJob() {
+    this.stopOnGoing = true;
+    this.preheatService.StopExecution({
+      projectName: this.projectName,
+      preheatPolicyName: this.preheatPolicyName,
+      executionId: +this.executionId,
+      execution: this.execution
+    })
+    .subscribe(response => {
+      this.stopOnGoing = false;
+       this.getExecutionDetail(true);
+       this.translate.get("REPLICATION.STOP_SUCCESS", { param: this.executionId }).subscribe((res: string) => {
+          this.messageHandlerService.showSuccess(res);
+       });
+    },
+    error => {
+      this.messageHandlerService.error(error);
+    });
+  }
+
+  viewLog(taskId: number | string): string {
+    return this.preheatService.rootUrl
+      + `/projects/${this.projectName}/preheat/policies/${this.preheatPolicyName}/executions/${this.executionId}/tasks/${taskId}/logs`;
+  }
+  clrLoadTasks(withLoading): void {
+      if (withLoading) {
+        this.loading = true;
+      }
+      this.preheatService.ListTasks({
+        projectName: this.projectName,
+        preheatPolicyName: this.preheatPolicyName,
+        executionId: +this.executionId
```

Comment:
Does it support backend paging?

Reply:
for task list, no!

Reply:
not support?

Reply:
![image](https://user-images.githubusercontent.com/30999793/88271587-f199a700-cd09-11ea-91b4-4849732ea197.png)